code/modules/antagonists/heretic/knowledge/void_lore.dm

```diff
@@ -42,7 +42,7 @@
 	)
 	required_atoms = list(/obj/item/knife = 1)
 	result_atoms = list(/obj/item/melee/sickly_blade/void)
-	limit = 2
+	limit = 20 //SKYRAT EDIT - ORIGINAL: 2
```

Comment:
5 at most, so it's not something they can afford to keep throwing away.

Reply:
Sure, I'll go with that for now.